Customer feedback updates (#1345)
auth0/docs-v2: main/docs/authenticate/identity-providers/social-identity-providers/devkeys.mdx

## Limitations of developer keys

> The Auth0 developer keys are intended for testing purposes, so there are a few caveats you need to be aware of when using them. These may cause your application to behave differently—or some functionality to not work at all—depending on whether you use your own Client ID and Client Secret, or whether you use the Auth0 developer keys.

Reviewer: ```suggestion
Auth0 developer keys are meant strictly for testing. Using them instead of your own Client ID and Client Secret can cause unexpected behavior, restricted functionality, or app errors.
```

When using the Auth0 developer keys, the authentication flow for the various identity providers may display Auth0's name, logo, and information to your users. When you register your own application, you have the opportunity to use your own logo and other application information instead.

<Frame>![Consent screen](/docs/images/cdy7uua7fh8z/5YcLIDCt3RaAkmK1cTeCJL/e85e16195e84aa52f7675faacd126853/consent-screen.png)</Frame>

## Limitations of developer keys when using Universal Login

If you are using the [Classic Login experience](/docs/authenticate/login/auth0-universal-login/universal-login-vs-classic-login/classic-experience) or the [Universal Login Experience](/docs/authenticate/login/auth0-universal-login/universal-login-vs-classic-login/universal-experience), the following limitations apply:

1. You cannot use developer keys with [custom domains](/docs/customize/custom-domains).

> 2. [Single Sign-On](/docs/authenticate/single-sign-on) will not function properly when using the Auth0 developer keys. The reason for this is that the Auth0 developer applications with all the relevant Identity Providers are configured to call back to the URL `https://login.auth0.com/login/callback` instead of the **callback URL** for your own tenant, for example `https://YOUR_AUTH0_DOMAIN/login/callback`.

Reviewer: ```suggestion
2. [Single Sign-On](/docs/authenticate/single-sign-on) will not function properly when using the Auth0 developer keys. The Auth0 developer applications are configured to call back to the URL `https://login.auth0.com/login/callback` instead of the **callback URL** for your own tenant, for example `https://YOUR_AUTH0_DOMAIN/login/callback`.
```

   This results in the SSO cookie not being set on your own tenant domain, so the next time a user authenticates, no SSO cookie will be detected, even if you configured your application to **Use Auth0 instead of the Identity Provider to do Single Sign-on** (legacy tenants only).

> 3. [Redirecting users with Actions](/docs/customize/actions/explore-triggers/signup-and-login-triggers/login-trigger/redirect-with-actions) will not function properly. This is because redirect Actions resume on the endpoint `https://YOUR_AUTH0_DOMAIN/continue`. When using Auth0's developer keys, the session is established on a special endpoint that is generic and tenant agnostic, and calling `/continue` will not find your previous session, resulting in an error.

Reviewer: ```suggestion
3. [Redirecting users with Actions](/docs/customize/actions/explore-triggers/signup-and-login-triggers/login-trigger/redirect-with-actions) will not function properly. Redirect Actions resume on the endpoint `https://YOUR_AUTH0_DOMAIN/continue`. When using Auth0's developer keys, the session is established on a special endpoint that is generic and tenant agnostic, and calling `/continue` will not find your previous session, resulting in an error.
```
